Design/DesignDoc.md

```diff
@@ -0,0 +1,220 @@
+# Design Document
+
+## _AmazonBudgetService_ Design
+
+## 1. Problem Statement
+
+Amazon Elite Budget Service is a highly advanced budgeting app that will enable users to have more control over their money/expenditures.
+Users can currently view their expenses by visiting the associated website/app for their card provider. However, users have requested that their expenditures and budget plans should be presented in a more streamlined, informative manner.
+
+This design document describes the main use cases and functionality of Amazon Elite Budget Service, a new, native AWS service that will provide users a cleaner, more informative means to view expenditures and budget goals.
+It is designed to interact with the Expense Management Client(which allows users to view expenditure information based on assigned tags and budget information based on their set goal).
```

Comment:
This sounds like the "Expense Management Client" is a completely different project, when I think it's probably the web ui/front-end for this project? For the context of the problem statement this is all one thing.